build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -0,0 +1,52 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines PyTorch models."""
+
+from e2e_test_framework import unique_ids
+from e2e_test_framework.definitions import common_definitions
+
+# Implementations of the models listed below can be found in `https://github.com/iree-org/iree-samples/tree/main/iree-torch/importer`.
+# We import the PyTorch models offline and make the .mlir available here for benchmarking.
+# If the mlir artifacts need to be updated, please run [update_torch_models.sh](https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh)
```

Comment:
The script at https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh could use more documentation

* Sample command line showing expected usage (including any CLI args or env vars)
* Prerequisites (Linux only? pip install or build some packages from source first? auth for gcloud)
* What the script does (how long it takes, how much disk/compute it needs, etc.)

Reply:
Created a PR with updates: https://github.com/iree-org/iree-samples/pull/111

I didn't include how long it takes and how much disk/compute it needs since it will change as we add more models.